nfts/src/tests/extrinsics.rs

```diff
@@ -9,375 +9,665 @@ fn origin(account: u64) -> mock::Origin {
 	RawOrigin::Signed(account).into()
 }
 
-/* fn root() -> mock::Origin {
+fn root() -> mock::Origin {
 	RawOrigin::Root.into()
-} */
+}
 
-mod transfer {
+mod create {
 	use super::*;
 
 	#[test]
-	fn cannot_transfer_delegated_nfts() {
-		ExtBuilder::new_build(vec![(ALICE, 100)]).execute_with(|| {
-			let nft_id = NFTs::create_nft(ALICE, vec![0], None).unwrap();
-			assert_ok!(NFTs::set_viewer(nft_id, Some(BOB)));
+	fn create_ok_with_no_series() {
+		ExtBuilder::default().caps(vec![(ALICE, 1000)]).build().execute_with(|| {
+			// State checks
+			assert_eq!(NFTs::nft_id_generator(), 0);
+			assert_eq!(NFTs::series_id_generator(), 0);
+
+			// Initial state
+			let alice: mock::Origin = origin(ALICE);
+			let serie_id = vec![50];
+			let ipfs_reference = vec![1];
+			let data = NFTData::new_default(ALICE, ipfs_reference.clone(), serie_id.clone());
+			let alice_balance = Balances::free_balance(ALICE);
 
-			let ok = NFTs::transfer(origin(ALICE), nft_id, BOB);
-			assert_noop!(ok, Error::<Test>::CannotTransferDelegatedNFTs);
+			// Create NFT with new serie id while there is no series already registered
+			assert_ok!(NFTs::create(
+				alice.clone(),
+				data.ipfs_reference.clone(),
+				Some(data.series_id.clone()),
+			));
+			let nft_id = NFTs::nft_id_generator() - 1;
+
+			// Events checks
+			assert_eq!(
```

Comment:
First we check the storage and then the event

Reply:
Do this for all other tests too.

Reply:
Done